src/GraphBLASExt/GxB_Clone.c

```diff
@@ -0,0 +1,43 @@
+/*
+* Copyright 2018-2019 Redis Labs Ltd. and Contributors
+*
+* This file is available under the Redis Labs Source Available License Agreement
+*/
+
+#include "GxB_Clone.h"
+#include <assert.h>
+
+GrB_Info GxB_MatrixClone
```

Comment:
It seems to me like `GrB_Matrix_dup` should accomplish the same as this?

Reply:
Ha! sorry I've missed it